docs/user-guide/src/version_support.md

```diff
@@ -81,13 +82,15 @@ The table describes which branches/image-tags are tested in each periodic CI tes
 | ------------------------------------------------ | ------------ | ----------- | -------------- | -------------- | ----------- | ---------- |
 | daily_main_integration_test_ubuntu/centos        | main         | main        | main           | latest         | latest      | latest     |
 | daily_main_e2e_integration_test_ubuntu/centos    | main         | main        | main           | latest         | latest      | latest     |
+| daily_release-1-6_integration_test_ubuntu/centos | release-1.6  | release-1.6 | release-0.5    | v0.5.0         | latest      | latest     |
 | daily_release-1-5_integration_test_ubuntu/centos | release-1.5  | release-1.5 | release-0.4    | v0.4.0         | latest      | latest     |
 | daily_release-1-4_integration_test_ubuntu/centos | release-1.4  | release-1.4 | release-0.3    | v0.3.1         | latest      | latest     |
 | daily_release-1-3_integration_test_ubuntu/centos | release-1.3  | release-1.3 | v0.2.0         | v0.2.0         | latest      | latest     |
```

Comment:
1.3 jobs are being dropped from the JJB right now, might as well drop them in the same PR.